app/src/main/java/com/randomappsinc/studentpicker/home/DeleteListDialog.java

```diff
@@ -19,14 +19,17 @@
     DeleteListDialog(@NonNull Listener listener, Context context) {
         dialog = new MaterialDialog.Builder(context)
                 .title(R.string.confirm_deletion_title)
-                .content(R.string.confirm_deletion_message)
                 .positiveText(android.R.string.yes)
                 .negativeText(android.R.string.cancel)
                 .onPositive((dialog, which) ->
                         listener.onDeleteListConfirmed(position)
                 ).build();
     }
 
+    void setContent(String content) {
+        dialog.setContent(content);
+    }
+
     public void show(int position) {
```

Comment:
Sorry for the miscommunication, I meant change this API to presentForList(int position, String listName). You can also do dialog.getContext() to get a context, and you can call getString from there. I think it's better to couple these 2 APIs together, since there isn't a use-case where you would call 1 without the other.